packages/nbdime/src/diff/widget/cell.ts

```diff
@@ -4,12 +4,16 @@
 
 import { Panel, Widget } from '@lumino/widgets';
 
+import { CodeEditor } from '@jupyterlab/codeeditor';
+
 import { IRenderMimeRegistry, MimeModel } from '@jupyterlab/rendermime';
 
 import { FlexPanel } from '../../upstreaming/flexpanel';
 
 import { CollapsiblePanel } from '../../common/collapsiblepanel';
 
+import { ICellDiffWidgetOptions } from '../../common/interfaces';
```

Comment:
Why is this not declared in this file? It seems specific to this widget rather than a common interface?

Reply:
It is also used here: `packages/nbdime/src/merge/widget/cell.ts`